frontend/testing/cypress/src/integration/studio/header.js

```diff
@@ -0,0 +1,34 @@
+import * as texts from '../../../../../language/src/nb.json';
+import { localChanges } from '../../selectors/localChanges';
+
+const designerAppId = `${Cypress.env('autoTestUser')}/${Cypress.env('designerAppName')}`;
+
+context('Header', () => {
+  before(() => {
+    cy.deleteAllApps(Cypress.env('autoTestUser'), Cypress.env('accessToken'));
+    cy.studioLogin(Cypress.env('autoTestUser'), Cypress.env('autoTestUserPwd'));
+    cy.createApp(Cypress.env('autoTestUser'), Cypress.env('designerAppName'));
+  });
+  beforeEach(() => {
+    cy.visit('/dashboard');
+    cy.visit('/editor/' + designerAppId);
```

Comment:
Jeg har opprettet en ny kommando som gjør dette uten at vi trenger å lage en `designerAppId`:
```suggestion
    cy.goToApp(Cypress.env('autoTestUser'), Cypress.env('designerAppName'));
```